main.go

```diff
@@ -200,12 +232,52 @@ func verticalScaling() error {
 	return nil
 }
 
-func getAWSClients() (*sqs.SQS, *rds.RDS, error) {
+func updateMemoryAlarm(client *cloudwatch.CloudWatch, alarmName, instanceClass string) error {
+	alarm, err := getExistingAlarm(client, alarmName, instanceClass)
+	if err != nil {
+		return errors.Wrap(err, "Failed to get existing Cloudwatch alarm")
+	}
+	_, err = client.PutMetricAlarm(&cloudwatch.PutMetricAlarmInput{
+		AlarmName:          aws.String(alarmName),
+		Metrics:            alarm.Metrics,
+		EvaluationPeriods:  alarm.EvaluationPeriods,
+		ComparisonOperator: alarm.ComparisonOperator,
+		Threshold:          alarm.Threshold,
+	})
```

Comment:
I'm a little confused. What are we updating here? It doesn't look like anything is changing

Reply:
@jwilander Maybe I need to rename the getExistingAlarm function because it returns the updated version of the alarm and not the existing one. In line 265 we are actually changing the only value that needs to be changed, which is the e1 metric.

Reply:
Ah, I see. Yeah if you could rename that function (and maybe combine this one into that one?) that would make that clearer

Reply:
@jwilander Renamed the functions, and moved some pieces around. Still kept the second function for easier error handling.